src/test/java/org/broadinstitute/consent/http/health/OntologyHealthCheckTest.java

```diff
@@ -36,11 +36,13 @@ public void setUp() {
     MockitoAnnotations.openMocks(this);
```

Comment:
Looks like this is a case where we didn't import the static in the original code, but we should.